bootstrap/httpserver/server.go

```diff
@@ -100,6 +105,12 @@ func report(reporting log.BasicLogger, h http.Handler) http.Handler {
 	})
 }
 
+func (s *server) dumpMetrics(w http.ResponseWriter, r *http.Request) {
```

Comment:
Couldn't find any tests for this.

Reply:
no tests :)
I don't see any value in testing this piece of code at this point